app/upgrade_handler.go

```diff
@@ -88,11 +66,19 @@ func (app ShentuApp) setUpgradeHandler() {
 			if !correctTypecast {
 				panic("mm.Modules[icatypes.ModuleName] is not of type ica.AppModule")
 			}
-
 			icamodule.InitModule(ctx, controllerParams, hostParams)
 
+			crisisGenesis := crisistypes.DefaultGenesisState()
+			crisisGenesis.ConstantFee.Denom = app.StakingKeeper.BondDenom(ctx)
+			app.CrisisKeeper.InitGenesis(ctx, crisisGenesis)
```

Comment:
Please correct me if I'm wrong. 
With this fix, the params will be all OK and we can have icamodule in the migration?

Reply:
yeap those three lines is just what crisis module migration does when it's first added to the modules